src/app/clusters/door-lock-server/door-lock-server.h

```diff
@@ -521,6 +548,60 @@ class DoorLockServer
     bool SetAttribute(chip::EndpointId endpointId, chip::AttributeId attributeId,
                       EmberAfStatus (*setFn)(chip::EndpointId endpointId, T value), T value);
 
+    /**
+     * @brief Reads non generic attributes for door lock.
+     *
+     * @param aPath         attribute path.
+     * @param aEncoder      attribute value encoder.
+     *
+     * @return CHIP_NO_ERROR  on success
+     * @return CHIP_ERROR     if attribute read failed
+     */
+    CHIP_ERROR Read(const chip::app::ConcreteReadAttributePath & aPath, chip::app::AttributeValueEncoder & aEncoder);
```

Comment:
Is this the AttributeAccessInterface implementation?  If so:

1. That should be documented; the rest of the documentation here would then be able to go away.
2. Why is it not marked `override`?

Reply:
Yes it is marked as override. removed comments